src/index.html

```diff
@@ -4,9 +4,340 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__icon">
+        <img src="images/logo.png" alt="icon">
+      </a>
+      <nav>
+        <ul class="header__nav">
+          <a href="#" class="header__link header__link--is-active is-active">
+            Apple
+          </a>
+          <a href="#" class="header__link">
+            Samsung
+          </a>
+          <a href="#" class="header__link">
+            Смартфоны и телефоны
+          </a>
+          <a
+            href="#"
+            data-qa="nav-hover"
+            class="header__link"
+          >
+            Ноутбуки и компьютеры
+          </a>
+          <a href="#" class="header__link">
+            Гаджеты
+          </a>
+          <a href="#" class="header__link">
+            Планшеты
+          </a>
+          <a href="#" class="header__link">
+            Фото
+          </a>
+          <a href="#" class="header__link">
+            Видео
+          </a>
+        </ul>
```

Comment:
Better add new block `nav` to `nav` tag, and list would be an element of nav block.

Please change styles according to changes in html.

And in `ul` should be `li` items.
```suggestion
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#Apple" class="nav__link nav__link--active is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#Samsung" class="nav__link">Samsung</a>
          </li>
          <li class="nav__item">
            <a href="#Phones" class="nav__link">Смартфоны и телефоны</a>
          </li>
          <li class="nav__item">
            <a
              href="#laptops"
              class="nav__link"
              data-qa="nav-hover"
            >
              Ноутбуки и компьютеры
            </a>
          </li>
```

Reply:
I fixed. But now the test Apple is failed, because I can't give the class "is active" to nav__item. If it's possible all will be correct.